migrations/001_init.sql

```diff
@@ -1,31 +1,35 @@
 -- +goose Up
 CREATE TABLE IF NOT EXISTS Chats (
-    chat_id INTEGER NOT NULL,
+    chat_id BIGINT NOT NULL,
     chat_type TEXT NOT NULL,
     token TEXT NOT NULL UNIQUE,
+    internal_id SERIAL PRIMARY KEY,
     CONSTRAINT unique_chat UNIQUE (chat_id, chat_type)
 );
 
 CREATE TABLE IF NOT EXISTS Subscriptions (
-    source_chat INTEGER NOT NULL,
-    destination_chat INTEGER NOT NULL,
-    FOREIGN KEY (source_chat) REFERENCES Chats(rowid),
-    FOREIGN KEY (destination_chat) REFERENCES Chats(rowid),
+    source_chat SERIAL NOT NULL,
```

Comment:
SERIAL is already not null. Btw, it should be integer here (and somewhere further) as SERIAL is not an actual datatype